tembo-cli/src/cmd/logs.rs

```diff
@@ -0,0 +1,117 @@
+use crate::cli::context::{get_current_context, Environment, Profile, Target};
+use anyhow::Result;
+use clap::Args;
+use std::process::Command;
+
+#[derive(Args)]
+pub struct LogsCommand {
+    #[clap(short, long)]
+    pub verbose: bool,
+}
+
+pub fn execute() -> Result<()> {
+    let instance_settings = super::apply::get_instance_settings(None)?;
+    let mut errors = Vec::new();
+    let env = get_current_context()?;
+
+    if env.target == Target::Docker.to_string() {
+        for (instance_name, _settings) in instance_settings {
+            match docker_logs(&instance_name) {
+                Ok(_) => (),
+                Err(e) => errors.push((instance_name, e)),
```

Comment:
What are we doing with this `errors` vec? Seems liek we should log the errors at some point

Reply:
we are printing out the logs too in docker_logs fn
```
let logs_stdout = String::from_utf8_lossy(&output.stdout);
    let logs_stderr = String::from_utf8_lossy(&output.stderr);
```